ee/vellum_ee/workflows/display/workflows/base_workflow_display.py

```diff
@@ -203,7 +203,7 @@ def serialize(self) -> JsonObject:
                     self.display_context.add_validation_error(validation_error)
 
                 serialized_node = node_display.serialize(self.display_context)
-            except (NotImplementedError, NodeValidationError, ValueError) as e:
+            except (NotImplementedError, NodeValidationError, InvalidInputReferenceError) as e:
```

Comment:
Have each of the custom exceptions extend `UserFacingException` and then only catch that one here